packages/kbn-text-based-editor/src/text_based_languages_editor.test.tsx

```diff
@@ -59,6 +59,21 @@ describe('TextBasedLanguagesEditor', () => {
     );
   }
   let props: TextBasedLanguagesEditorProps;
+  beforeAll(() => {
```

Comment:
I don't understand why we need to add this here 🤔

Reply:
Thanks for the review @stratoula! After I moved the code editor theme to the package, this and one more test started failing with an error message `TypeError: window.matchMedia is not a function`. I suppose the test started failing because the tested component uses `CodeEditor` but I'm not entirely sure how `matchMedia` is related to this. 🤔 

I found [this forum](https://stackoverflow.com/questions/39830580/jest-test-fails-typeerror-window-matchmedia-is-not-a-function) where people suggest mocking `matchMedia` (there is also [Jest documentation](https://jestjs.io/docs/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom) for this). This fixed the failures, and I noticed that this approach is also used in other tests in Kibana (e.g. 
in a [session_view test](https://github.com/elastic/kibana/blob/2bd49f5ae4816c75a87a371e58ca2b7ea73d4bd4/x-pack/plugins/session_view/public/components/tty_player/hooks.test.tsx#L19)). I added a comment to explain why this is there, sorry for not clarifying in the PR!

Reply:
Weird but thanx for the explanation Elena. I will wait for the CI to get green and finish with my review :)

Reply:
Probably can be moved to global jest setup, I'll give it a try